install-steps.md

```diff
@@ -618,6 +618,144 @@ podman run -d --name rhcos_image_cache \
 registry.centos.org/centos/httpd-24-centos7:latest
 ~~~
 
+## Create a Disconnected Registry (optional)
```

Comment:
This whole section likely needs to go under `Configure the install-config and metal3-config` because there are pieces that you append to the install-config.yaml but that file does not exist yet.

Reply:
Would that break the flow and make things confusing if not using disconnected registry?
Would it be better to move this to its own section at the end of the document and place a note between steps 1 and 2 of the "## Configure the install-config and metal3-config" section referring to this if they wish to do disconnected registry at the end of the document?

Reply:
this has been addressed and pushed in latest commmit.